examples/cpp/hello_world/PublisherApp.cpp

```diff
@@ -135,6 +136,15 @@ void PublisherApp::run()
         {
             std::cout << "Message: '" << hello_.message() << "' with index: '" << hello_.index()
                       << "' SENT" << std::endl;
+
+            if (hello_.index() == 1u)
+            {
+                ReturnCode_t acked = RETCODE_ERROR;
+                do {
+                    dds::Duration_t acked_wait{1, 0};
+                    acked = writer_->wait_for_acknowledgments(acked_wait);
+                }while (acked != RETCODE_OK);
```

Comment:
```suggestion
                do
                {
                    dds::Duration_t acked_wait{1, 0};
                    acked = writer_->wait_for_acknowledgments(acked_wait);
                } while (acked != RETCODE_OK);
```

Reply:
Now that we are testing the examples, it would be nice to check uncrustify on their code (in separate PR, of course)

Reply:
Good idea, yes, opening in a few moments